### PR TITLE
data displayed

### PR DESCRIPTION
**Pull request by supernova-97**
Inputs from the form will be displayed on the dashboard

### CODE REVIEW

File: components/Dashboard/index.js [hunk 1]
```diff
@@ -7,7 +7,19 @@ function Dashboard({ tripData }) {
...
+            <DetailedSection>
+              <TripInfoHeading>Trip name:</TripInfoHeading>{" "}
+              <TripInfo>{tripData.name}</TripInfo>
+              <TripInfoHeading>Where to:</TripInfoHeading>{" "}
+              <TripInfo>{tripData.where}</TripInfo>
+              <TripInfoHeading>Date:</TripInfoHeading>{" "}
+              <TripInfo>{tripData.when}</TripInfo>
+              <TripInfoHeading>Companions:</TripInfoHeading>{" "}
+              <TripInfo>{tripData.who.join(", ")}</TripInfo>
+            </DetailedSection>
```
Comment (PReichetanz) [inline components/Dashboard/index.js]: To be more semantic, I would reconsider the HTML elements and use a [description list `<dl>` ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) here instead. The [heading will be `<dt>` ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt) elements and [the content `<dd>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd).

You can ask Melissa how to implement these elements 😉

**Review by PReichetanz — CHANGES_REQUESTED** [review]
Nice work, but I would suggest to use more semantic HTML elements here.

File: components/Dashboard/index.js (hunk 1 above)
Comment (supernova-97) [reply components/Dashboard/index.js]: I didn't know these existed, but they seem to fit perfectly! I'll change it (or ask Melissa how to do it haha)

**Review by PReichetanz — APPROVED** [review]
LGTM

**Pull request merged**